egs/heroico/s5/run.sh

```diff
@@ -12,72 +12,53 @@ set -o pipefail
 set -u
 
 # the location of the LDC corpus; this location works for the CLSP grid.
-datadir=/export/corpora5/LDC/LDC2006S37
+#datadir=/export/corpora5/LDC/LDC2006S37
```

Comment:
The data directory should always be defined in the top-level run.sh, not in scripts in local/, so people can see it more easily and change it.  This should be an argument to the data-preparation script.  Same for the lexicon location: better to have it defined inthe top-level script so that it's more visible.